packages/apidom-ls/src/config/openapi/responses/completion.ts

```diff
@@ -23,211 +23,266 @@ const httpCode3_0CompletionRule = {
   ],
 };
 
+// eslint-disable-next-line @typescript-eslint/naming-convention
+const httpCode3_1CompletionRule = {
+  kind: 14,
+  format: CompletionFormat.OBJECT,
+  type: CompletionType.PROPERTY,
+  insertTextFormat: 2,
+  documentation: {
+    kind: 'markdown',
+    value:
+      '[Response Object](https://github.com/OAI/OpenAPI-Specification/blob/main/versions/3.1.0.md#responseObject) \\| [Reference Object](https://github.com/OAI/OpenAPI-Specification/blob/main/versions/3.1.0.md#referenceObject)\n\\\n\\\nThe documentation of responses other than the ones declared for specific HTTP response codes. Use this field to cover undeclared responses.',
+  },
+  targetSpecs: [{ namespace: 'openapi', version: '3.1.0' }],
+};
+
 const completion: ApidomCompletionItem[] = [
   {
```

Comment:
You cannot do this, you need to duplicate every rules. After this change things will work as expected

```s
  {
    label: 'default',
    insertText: 'default',
    ...httpCode3_0CompletionRule,
  } as ApidomCompletionItem,
  {
    label: 'default',
    insertText: 'default',
    ...httpCode3_1CompletionRule,
  } as ApidomCompletionItem,